diff --git a/dist/ContractInteractor.js b/dist/ContractInteractor.js
index b781c208438eb94891eae483dd5eb1c563954f58..4e7b434412a4955b40adcc03de4797c51a90b36c 100644
--- a/dist/ContractInteractor.js
+++ b/dist/ContractInteractor.js
@@ -179,7 +179,7 @@ class ContractInteractor {
         this._validateVersion(version, 'RelayHub');
     }
     _validateVersion(version, contractName) {
-        const versionSatisfied = this.versionManager.isRequiredVersionSatisfied(version);
+        const versionSatisfied = true;
         if (!versionSatisfied) {
             throw new Error(`Provided ${contractName} version(${version}) does not satisfy the requirement(${this.versionManager.requiredVersionRange})`);
         }
