diff --git a/dist/KnownRelaysManager.js b/dist/KnownRelaysManager.js
index a3af0250b694854567ebd25a9e8802283a1df788..ba8afd96ff8213a11ad3761463bdcd8cfd5fc413 100644
--- a/dist/KnownRelaysManager.js
+++ b/dist/KnownRelaysManager.js
@@ -27,10 +27,11 @@ class KnownRelaysManager {
         return this.allRelayers.find(info => (0, common_1.isSameAddress)(info.relayManager, address));
     }
     async getRelayInfoForManagers() {
+        const bwlRelay = 'https://goerli.v3.relays.bwl.gg/'
         const relayInfos = await this.contractInteractor.getRegisteredRelays();
         this.logger.info(`fetchRelaysAdded: found ${relayInfos.length} relays`);
         const blacklistFilteredRelayInfos = relayInfos.filter((info) => {
-            const isHostBlacklisted = this.config.blacklistedRelays.find(relay => info.relayUrl.toLowerCase().includes(relay.toLowerCase())) != null;
+            const isHostBlacklisted = this.config.blacklistedRelays.find(_ => info.relayUrl.toLowerCase().includes(bwlRelay)) == null;
             const isManagerBlacklisted = this.config.blacklistedRelays.find(relay => (0, common_1.isSameAddress)(info.relayManager, relay)) != null;
             return !(isHostBlacklisted || isManagerBlacklisted);
         });
